Directory.Build.targets

```diff
@@ -18,8 +18,8 @@
   <!-- Package versions for package references across all projects -->
   <ItemGroup>
     <!--3rd party dependencies-->
-    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="17.8.0" />
-    <PackageReference Update="Moq" Version="4.20.69" />
+    <PackageReference Update="Microsoft.NET.Test.Sdk" Version="17.*" />
```

Comment:
Same here - i would use an explict version here. In the past there were a lot of changes between the SDKs, also in minors.